tools/README.md

```diff
@@ -0,0 +1,79 @@
+# prepare_lava_job
+
+prepare_lava_job is a Python script designed to generate Job files in YAML format. These job files are designed to be used with LAVA in order to automate the process of flashing and testing RevPi devices, thus saving time in identifying errors in the images.
+
+Once the LAVA Job has been generated, it can be submitted using one of the following methods:
+- lavacli command-line tool
+- LAVA webUI
+- LAVA REST/XML-RPC API
+
+Please refer to the LAVA documentation for detailed instructions on how to submit the generated job. For a example using lavacli, see the "Example" section.
+
+## Features
+
+- Generates LAVA Job files for flashing and testing images on RevPi devices.
+- Allows customization of job parameters through command-line arguments.
+
+## Usage
+
+The LAVA Job Generator can be executed using the following command:
+
+- -d, --device-type: Specify the device type to use for testing.
+- -u, --url: Provide the URL of the image to be flashed.
+- --flash: Specify the manifest used to flash the given image onto the DUT (required).
+- -s, --testsuite: Specify the job testsuite to be used after flashing (required).
+- -t, --tags: Specify the tag used for selecting an appropriate device for testing.
+
+> **_NOTE:_**  By default, the generated job is configured for a RevPi Core device (device_type: RevPi_Core) and uses the "config_core_01" tag.
+
+> **_NOTE:_**  The default image is "2023-06-26-revpi-bullseye-rc3-arm64-lite.img". 
+
+> **_NOTE:_**  The default behavior can be modified by providing appropriate options.
+---
+
+- A default job used to flash the image can be found here: [../automated/deployment/job-flash-RevPi_Core.yaml](../automated/deployment/job-flash-RevPi_Core.yaml)
+
+- A testsuite with a compilation of tests for RevPi Core devices can be found here: [../automated/testsuites/core-testsuite-001.yml](../automated/testsuites/core-testsuite-001.yml)
+
+---
+
+For more information, please consult the LAVA documentation and the examples provided in this repository.
+
+### Output
+
+```
+usage: prepare_lava_job [-h] [-d [DEVICE_TYPE]] [-u URL] --flash [FLASH] -s TESTSUITE [-t [TAGS]]
+
+This script is designed to generate a LAVA Job-file in yaml, which can be used with our RevPi devices in order to
+flash and perform tests with LAVA automatically, thus saving time finding errors in the images.
+
+options:
+  -h, --help            show this help message and exit
+  -d [DEVICE_TYPE], --device-type [DEVICE_TYPE]
+                        Device type to use for testing
+  -u URL, --url URL     URL of the image to be flashed
+  --flash [FLASH]       Manifest used to flash the given image onto the DUT
+  -s TESTSUITE, --testsuite TESTSUITE
+                        Job testsuite to be used after flashing
+  -t [TAGS], --tags [TAGS]
+                        Tag used for selecting an appropriate device for testing
+
+Default behaviour: Generate a LAVA Job to flash and run tests on a given RevPi device.
+```
```

Comment:
I'm not sure if this has any use being in the README.md. The script can be executed without compilation to get the help output and the description of the tool as well as the description of the options has already been provided in the README :)

Reply:
And I think you are right. If you modify something in the structure, you have to modify a lot more having this here.... I'm going to delete it..